src/components/forums/topic/PostContent.tsx

```diff
@@ -485,7 +507,7 @@ export function PostContent(props: PostContentProps) {
                             block: "center",
                           });
                         }}>
-                        Reply <Reply />
+                        {windowSize > 768 ? "Reply" : ""} <Reply />
```

Comment:
same as above

Reply:
Yes, it's possible to hide the text of the buttons using only css